cdm/core/src/main/java/ucar/nc2/filter/ConvertMissing.java

```diff
@@ -189,12 +187,7 @@ public boolean isMissingValue(double val) {
     if (!hasMissingValue) {
       return false;
     }
-    for (double aMissingValue : missingValue) {
-      if (Misc.nearlyEquals(val, aMissingValue, Misc.defaultMaxRelativeDiffFloat)) {
-        return true;
-      }
-    }
-    return false;
+    return Arrays.stream(missingValue).anyMatch(mVal -> Misc.nearlyEquals(val, mVal, Misc.defaultMaxRelativeDiffFloat));
```

Comment:
this one could also use `DoubleMath.fuzzyEquals`

Reply:
well I replace the implementation of `Misc.nearlyEquals` with 'DoubleMath.fuzzyEquals`

Reply:
oops, missed that!